dev-site-config/site.config.js

```diff
@@ -0,0 +1,15 @@
+const navConfig = require('./navigation.config');
+
+const siteConfig = {
+  appConfig: {
+    themes: {
+      'Default Theme': '',
```

Comment:
Since this is for toolkit, can we only run one theme?

Reply:
nah, we can run all the themes, but toolkit itself will never update themes. Might be nice for testing the functional testing library